src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3217,6 +3217,158 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshotAccessRequests/{id}/members:
+    get:
+      tags:
+        - SnapshotAccessRequest
+      description: Get the members in the DAC group
+      operationId: getSnapshotAccessRequestGroupMembers
+      parameters:
+        - $ref: '#/components/parameters/Id'
+      responses:
+        200:
+          description: The group members of this request has been fetched
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SnapshotAccessRequestMembersResponse'
+        400:
+          description: Bad request - invalid id, snapshotRequest, or badly formed IamResourceType.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        401:
+          description: Unauthorized - dataset does not exist or missing permissions to view.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        403:
+          description: Unauthorized - missing permissions to request snapshot.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        404:
+          description: The group members cannot be found or you do not have access
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        500:
+          description: An unexpected error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+    post:
+      tags:
+        - SnapshotAccessRequest
+      description: Adds a member to the DAC group
+      operationId: addSnapshotAccessRequestGroupMember
+      parameters:
+        - $ref: '#/components/parameters/Id'
+        - name: memberEmail
+          in: query
+          description: The email of the user to add
+          required: true
+          schema:
+            type: string
```

Comment:
It's confusing to me that this API takes both `memberEmail` and `PolicyMemberRequest`, I would expect the API to only accept one or the other. Is there a need for both in this API? If you only need one, I would use `requestBody`, as that is more typical for `POST` than `query`.